esmvaltool/cmorizers/obs/cmorize_obs_phc.py

```diff
@@ -0,0 +1,124 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer for PHC data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   http://psc.apl.washington.edu/nonwp_projects/PHC/Data3.html
+
+Last access
+   20190131
+
+Go to the `DOWNLOAD DATA (NetCDF)`, and download `ANNUAL` fields.
```

Comment:
```suggestion
Go to `DOWNLOAD DATA (NetCDF)` and download the `ANNUAL` fields 
for both `TEMPERATURE` and `SALINITY`.
```

Reply:
You should also add that the files have to be extracted (or let the script automatically do this).

Reply:
Sorry, please ignore the above comments.
I've been downloading the ASCII version.